lib/strvals/parser.go

```diff
@@ -1,9 +1,10 @@
-package log
+package strvals
 
 import "fmt"
 
-type token struct {
-	key, value string
+// Token represents a key-value token.
+type Token struct {
```

Comment:
After the renaming to `strvals` package and `Parse` method, I left this as `Token` as the definition has not changed, but I could also see this and `tokenizer` changing to:
```
s/Token/KV
s/tokenizer/parser
```

Reply:
The latter can definitely be changed later as it is not exported.

I am don't think `Token` is not specific enough - so it is okay IMO.